web/vtadmin/src/components/ActionPanel.test.tsx

```diff
@@ -33,11 +33,7 @@ describe('ActionPanel', () => {
      * provides such a function and should be `render`ed in the context QueryClientProvider.
      */
     const Wrapper: React.FC<ActionPanelProps & { url: string }> = (props) => {
-        const mutation = useMutation(() => fetch(new URL(props['url']), { method: 'post' }), {
-            onError: (error) => {
-                console.log('ERROR: ', error);
-            },
-        });
+        const mutation = useMutation(() => fetch(new URL(props['url']), { method: 'post' }));
```

Comment:
Was the removal of the console.log for the error intentional? I am guessing this only impacts our own debugging on vtadmin.

Reply:
Yeah - I removed it because I accidentally left it in there!